receiver/kubeletstatsreceiver/kubelet/mem.go

```diff
@@ -38,3 +41,15 @@ func memUsageMetric(prefix string, s *stats.MemoryStats) *metricspb.Metric {
 func memRssMetric(prefix string, s *stats.MemoryStats) *metricspb.Metric {
 	return intGauge(prefix+"memory.rss", "By", s.RSSBytes)
 }
+
+func memWorkingSetMetric(prefix string, s *stats.MemoryStats) *metricspb.Metric {
+	return intGauge(prefix+"memory.workingset", "By", s.WorkingSetBytes)
+}
+
+func memPageFaultsMetric(prefix string, s *stats.MemoryStats) *metricspb.Metric {
+	return intGauge(prefix+"memory.pagefaults", "1", s.PageFaults)
+}
+
+func memMajorPageFaultsMetric(prefix string, s *stats.MemoryStats) *metricspb.Metric {
+	return intGauge(prefix+"memory.majorpagefaults", "1", s.MajorPageFaults)
```

Comment:
I think we should make more readable: `memory.major_page_faults`. Here is an example in conventions https://github.com/open-telemetry/oteps/blob/master/text/0119-standard-system-metrics.md#systemswap

Reply:
Agree, fixed.